nodejs/week1/avg.js

```diff
@@ -6,13 +6,10 @@ if (args.length == 0) {
 }
 
 const numbers = args.map(Number);
-
 if (numbers.some(isNaN)) {
   console.log("All arguments must be valid numbers.");
```

Comment:
I would suggest to use Number.isNaN here . 
https://stackoverflow.com/questions/33164725/confusion-between-isnan-and-number-isnan-in-javascript